File: exercises/beecrowd/1098/1098.go
// https://judge.beecrowd.com/en/problems/view/1098
package main

import "fmt"

func format(n int) string {
	if n%10 == 0 {
		return fmt.Sprintf("%d", n/10)
	}
	return fmt.Sprintf("%.1f", float64(n)/10)
}

func main() {
	for i := 0; i <= 20; i += 2 {
		for j := 1; j <= 3; j++ {
			fmt.Printf("I=%s J=%s\n", format(i), format(i+10*j))
		}
	}
}

File: exercises/beecrowd/1098/1098.java
// https://judge.beecrowd.com/pt/problems/view/1098
public class Main {
    public static void main(String[] args) {

        double I = 0;

        while (I <= 2.0 + 1e-9) {
            for (double J = 1; J <= 3; J++) {

                double vI = Math.round(I * 10.0) / 10.0;
                double vJ = Math.round((J + I) * 10.0) / 10.0;

                if (vI == (int) vI) {
                    System.out.println("I=" + (int)vI + " J=" + (int)vJ);
                } else {
                    System.out.printf("I=%.1f J=%.1f\n", vI, vJ);
                }
            }
            I += 0.2;
        }
    }
}
